Source/UINavigation/Public/UINavWidget.h

```diff
@@ -31,19 +31,24 @@ class UINAVIGATION_API UUINavWidget : public UUserWidget
 	bool bCompletedSetup = false;
 	bool bSetupStarted = false;
 
-	bool bShouldTick = true;
 	bool bMovingSelector = false;
 	bool bIgnoreMouseEvent = false;
 	bool bReturning = false;
+		
+	bool bUINavSetup = false;
```

Comment:
Rename this to something like bShouldTickUINavSetup

Reply:
ok